encryption/encryptor.go

```diff
@@ -123,7 +122,8 @@ func (s *encryptor) ConfirmMessageProcessed(messageID []byte) error {
 	confirmationData, ok := s.messageIDs[id]
 	if !ok {
 		s.logger.Debug("could not confirm message", zap.String("messageID", id))
-		return fmt.Errorf("message with ID %#x not found", messageID)
+		// We are ok with this, means no key material is stored (public message, or already confirmed)
```

Comment:
Should the log message reflect it then (e.g. `could not confirm message or message already confirmed`, or something like that)?